src/main/java/org/jitsi/videobridge/rest/ColibriWebSocketService.java

```diff
@@ -160,7 +161,7 @@ ServletHolder initializeColibriWebSocketServlet(
         ServletHolder holder = null;
 
         ConfigurationService cfg
-            = ServiceUtils.getService(bundleContext, ConfigurationService.class);
+            = ServiceUtils2.getService(bundleContext, ConfigurationService.class);
```

Comment:
what is `ServiceUtils2` vs `ServiceUtils`?

Reply:
`ServiceUtils` comes from the `jitsi-utils` bundle (I think. It might be another one of the jitsi bundles). `ServiceUtils2` is it's replacement in `jicoco`.